### PR TITLE
STORM-847 Adding new cli for getting latest errors from running storm topology

### PR DESCRIPTION
**Pull request by snikhil5**
This has been tested with the latest src code.

### CODE REVIEW

File: storm-core/src/clj/backtype/storm/command/get_errors.clj [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          topologies (.get_topologies cluster-info)
+          topo-id (get-topology-id name topologies)
+          topo-info (when (not-nil? topo-id) (.getTopologyInfoWithOpts nimbus topo-id opts))]
+      (if (or (nil? topo-id) (nil? topo-info))
+        (println (to-json {"Failure" (str "No topologies running with name " name)}))
+        (let [topology-name (.get_name topo-info)
+              topology-errors (.get_errors topo-info)]
+          (println  (to-json (hash-map
+                                   "Topology Name" topology-name
+                                   "Comp-Errors" (get-component-errors topology-errors)))))))))
```
Comment (d2r) [inline storm-core/src/clj/backtype/storm/command/get_errors.clj]: really minor: There are some strange formatting issues in the last three lines.

File: bin/storm.py [hunk 2]
```diff
@@ -304,6 +304,19 @@ def rebalance(*args):
...
+
+    Get the latest error from the running topology. The returned result contains
+    the key value pairs for component-name and component-error for the components in error.
+    The result is returned in json format.
+    """
+    exec_storm_class(
+        "backtype.storm.command.get_errors",
+        args=args,
+        jvmtype="-client",
+        extrajars=[USER_CONF_DIR, STORM_DIR + "/bin"])
```
Comment (d2r) [inline bin/storm.py]: Let's use `os.path.join(STORM_DIR, "bin")` here.

**Comment by d2r** [comment]
Looks OK to me overall.  Just minor comments.

**Comment by snikhil5** [comment]
updated..

**Comment by d2r** [comment]
+1 looks good

**Comment by knusbaum** [comment]
+1

**Comment by kishorvpatil** [comment]
Thank you @snikhil5. This is pulled into master.

**Pull request merged**